src/map/lua/luautils.cpp

```diff
@@ -3924,11 +3924,11 @@ namespace luautils
     {
         DropList_t* DropList = itemutils::GetDropList((uint16)lua_tointeger(L, 1));
```

Comment:
Can we add a pointer guard here? It currently crashes if called with an invalid Drop ID. Not your fault, just something to clean up while you're in here. :)

Reply:
Sure, I can do that.

Reply:
Fixed this in my second commit